static/game/bullet.js

```diff
@@ -128,6 +131,9 @@ export class Bullet extends Entity {
     // remove the bullet if it's not supposed to bounce
     if (!this.reflectsOffWalls) {
       this.deleteMe = true;
+    } else {
+      this.reflectNum--;
```

Comment:
Why do we need a number for the amount of times its reflected?

Reply:
Old debug feature, never removed.